From 8a34ddc46e4c42cbfec486b9c857af2ccba00176 Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Tue, 13 Aug 2013 13:55:47 +0200
Subject: [RHEL6 LIBREPORT PATCH 106/106] rhbz: get id of duplicate from
 correct field

This problem was found in dump directories uploaded by Emergency
analysis

Related to rhbz#991088

Signed-off-by: Jakub Filak <jfilak@redhat.com>
Signed-off-by: Martin Milata <mmilata@redhat.com>
Signed-off-by: Jiri Moskovcak <jmoskovc@redhat.com>
---
 src/plugins/rhbz.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/rhbz.c b/src/plugins/rhbz.c
index 9283b81..f526eac 100644
--- a/src/plugins/rhbz.c
+++ b/src/plugins/rhbz.c
@@ -545,7 +545,7 @@ struct bug_info *rhbz_bug_info(struct abrt_xmlrpc *ax, int bug_id)
     if (strcmp(bz->bi_status, "CLOSED") == 0 && !bz->bi_resolution)
         error_msg_and_die(_("Bug %i is CLOSED, but it has no RESOLUTION"), bz->bi_id);
 
-    ret = (int*)rhbz_bug_read_item("dup_id", bug_item,
+    ret = (int*)rhbz_bug_read_item("dupe_of", bug_item,
                                    RHBZ_READ_INT);
     if (strcmp(bz->bi_status, "CLOSED") == 0
         && strcmp(bz->bi_resolution, "DUPLICATE") == 0
-- 
1.8.3.1

